src/extension.ts

```diff
@@ -258,6 +258,27 @@ class ClientWorkspace {
             wslPath = wslPath.substr('file://'.length);
           }
 
+          wslPath = wslPath.split('%20').join(' ');
+          wslPath = wslPath.split('%21').join('!');
+          wslPath = wslPath.split('%23').join('#');
+          wslPath = wslPath.split('%24').join('$');
+          wslPath = wslPath.split('%26').join('&');
+          wslPath = wslPath.split('%27').join('\'');
+          wslPath = wslPath.split('%28').join('(');
+          wslPath = wslPath.split('%29').join(')');
+          wslPath = wslPath.split('%2B').join('+');
+          wslPath = wslPath.split('%2C').join(',');
+          wslPath = wslPath.split('%2D').join('-');
+          wslPath = wslPath.split('%2E').join(',');
+          wslPath = wslPath.split('%3D').join('=');
+          wslPath = wslPath.split('%40').join('@');
+          wslPath = wslPath.split('%5B').join('[');
+          wslPath = wslPath.split('%5D').join(']');
+          wslPath = wslPath.split('%5E').join('^');
+          wslPath = wslPath.split('%7B').join('{');
+          wslPath = wslPath.split('%7D').join('}');
+          wslPath = wslPath.split('%7E').join('~');
```

Comment:
For future reference, while this looks declarative (although it's not immediately obvious it's a replace operation), this might not be good from the performance standpoint. Always profile first, but doing this many string manipulations (and separately, one after another) looks like an unnecessary performance overhead